set_env_vars-sample.sh

```diff
@@ -19,4 +19,8 @@ export DYNAMO_DB_URL='http://dynamodb:8000'
 # Salesforce
 export SALESFORCE_AUTH_URL='https://test.salesforce.com'
 export SALESFORCE_CLIENT_ID=''
-export SALESFORCE_CLIENT_SECRET=''
\ No newline at end of file
+export SALESFORCE_CLIENT_SECRET=''
+
+# Local Cache
+export LOCAL_CACHE_HOST='localhost'
```

Comment:
Why are we calling it local?

Reply:
For local only we are using cache host and port, for dev and prod we will be using aws creds